content/kubernetes/reference/supported_k8s_distributions.md

```diff
@@ -140,50 +146,71 @@ For details on this platform, see the [GKE documentation](https://cloud.google.c
+| **1.23** (2.6) | <span title="Deprecated">&#x26A0;&#xFE0F;</span> | <span title="Deprecated">&#x26A0;&#xFE0F;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |  |  |  |  |  |  |
+| **1.22** | <span title="Deprecated">&#x26A0;&#xFE0F;</span> | <span title="Deprecated">&#x26A0;&#xFE0F;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |  |  |  |  |
+| **1.21** |  | <span title="X icon">&#x274c;</span> | <span title="Deprecated">&#x26A0;&#xFE0F;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |  |  |  |  |
+| **1.20** |  |  |  |  |  | <span title="X icon">&#x274c;</span> | <span title="Deprecated">&#x26A0;&#xFE0F;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |  |  |
+| **1.19** |  |  |  |  |  | <span title="X icon">&#x274c;</span> | <span title="Deprecated">&#x26A0;&#xFE0F;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |  |  |
+| **1.18** |  |  |  |  |  |  |  | <span title="X icon">&#x274c;</span> | <span title="Deprecated">&#x26A0;&#xFE0F;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |
+| **1.17** |  |  |  |  |  |  |  | <span title="X icon">&#x274c;</span> | <span title="Deprecated">&#x26A0;&#xFE0F;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |
+| **1.18** |  |  |  |  |  |  |  |  |  | <span title="X icon">&#x274c;</span> | <span title="Deprecated">&#x26A0;&#xFE0F;</span> | <span title="Supported">&#x2705;</span> |
+| **1.17** |  |  |  |  |  |  |  |  |  | <span title="X icon">&#x274c;</span> | <span title="Deprecated">&#x26A0;&#xFE0F;</span> | <span title="Supported">&#x2705;</span> |
 
 {{</table-scrollable>}}
 
 ## VMware Tanzu Kubernetes Grid (TKG)
 
 For details on this platform, see the [TKG documentation](https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/index.html).
 
+#### TKG 1.22
+
+{{<table-scrollable>}}
+|  | **<nobr>**7.2.4**</nobr>** | **<nobr>**6.4.2-6**</nobr>** | **<nobr>**6.4.2-5**</nobr>** | **<nobr>**6.2.18-41**</nobr>** |
+|---|---|---|---|---|
+| **1.24** | <span title="Supported">&#x2705;</span> |  |  |  |
+|**1.23** | <span title="Supported">&#x2705;</span> |  |  |  |
+| **1.22** | <span title="Supported">&#x2705;</span> |  |  |  |
+{{</table-scrollable>}}
+
+#### TKG 1.21
```

Comment:
I believe TKG 1.21/1.22 support hasn't been completed yet, and I'm not sure it'll complete before Quartz goes public. AFAIK the intent is to release without support statements for 1.21/1.22 and update that when support is established.

Also, TKG 1.21 only support k8s 1.24 (and not 1.22/1.23) and we'll only certify it on k8s 1.24 accordingly.
https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/2.1/tkg-deploy-mc-21/mgmt-release-notes.html#supported-kubernetes-versions-in-tanzu-kubernetes-grid-v21-5

Reply:
As far as I know yes, but @yuvallevy2 can you keep me honest on this?